package.json

```diff
@@ -50,7 +50,7 @@
     "core-object": "^1.1.0",
     "ember-cli-babel": "^5.0.0",
     "ember-cli-deploy-plugin": "^0.2.2",
-    "lodash": "^3.9.3",
+    "lodash": "^4.11.1",
```

Comment:
@ghedamat No particular reason but to keep up-to-date. I wasn't aware of any recent problems with lodash, I can revert it back to 3 no problems.